src/Factories/Logger.php

```diff
@@ -27,14 +27,14 @@ public function __construct(
     }
 
     /**
-     * @param  array<string, mixed>  $config
+     * @param  array<string, mixed>&array{level?: \Psr\Log\LogLevel::*}  $config
      */
     public function __invoke(array $config): LoggerInterface
     {
         return new Monolog(
             name: 'nightwatch',
             handlers: [
-                new LogHandler($this->nightwatch),
+                new LogHandler($this->nightwatch, Monolog::toMonologLevel($config['level'] ?? 'debug')),
```

Comment:
We can provide the default in the `registerLogger` method.

```suggestion
                new LogHandler($this->nightwatch, Monolog::toMonologLevel($config['level'])),
```